Rework bootstrapping docs (#2129)
chef/chef-web-docs: chef_master/source/chef_overview.rst

     - .. tag node

       A node is any device---physical, virtual, cloud, network device, etc.---that is under management by Chef Infra.

       .. end_tag

>        A Chef Infra Client is installed on each node that is under management by Chef Infra. The Chef Infra Client performs all of the configuration tasks that are specified by the run-list and will pull down any required configuration data from the Chef Infra Server as it is needed during a Chef Infra Client run.

Reviewer: ```suggestion
       Chef Infra Client is installed on each node that is managed with Chef Infra. Chef Infra Client configures the node locally by performing the tasks specified in the run-list. Chef Infra Client will also pull down any required configuration data from the Chef Infra Server during a Chef Infra Client run.
```
